adafruit_esp32spi/adafruit_esp32spi_socketpool.py

```diff
@@ -60,12 +60,12 @@ def getaddrinfo(  # pylint: disable=too-many-arguments,unused-argument
         if not isinstance(port, int):
             raise ValueError("Port must be an integer")
         ipaddr = self._interface.get_host_by_name(host)
-        return [(SocketPoolContants.AF_INET, socktype, proto, "", (ipaddr, port))]
+        return [(SocketPoolConstants.AF_INET, socktype, proto, "", (ipaddr, port))]
 
     def socket(  # pylint: disable=redefined-builtin
         self,
-        family=SocketPoolContants.AF_INET,
-        type=SocketPoolContants.SOCK_STREAM,
+        family=SocketPoolConstants.AF_INET,
```

Comment:
@dhalbert I'm open to suggestions, but this is why the superclass was added. Since we are in the class, it's not defined and thus this line will error on import:
<img width="494" alt="image" src="https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/assets/1547881/2798dbe6-ef5d-4b80-8d9a-eb45ffed5c08">
_ignore the line number in the image, it's this one_

Reply:
I see. But what about something like this? This works:
```py
class ConstantsTest:
    A = 1
    B = 2

    def f(self, a=A, b=B):
        pass
```
The unqualified name works.

This actually is more complicated. In CPython, `AF_INET`, etc. are defined at the module level, not inside the `Socket` class. I can't remember why we defined them as class constants in `SocketPool` instead of in `socketpool`.

Reply:
Ahh, I can do a commination of both...

I'm assuming they are in `SocketPool`, since that's where they were when ESP32SPI was created and so when `socketpool.SocketPool` was created if followed suit?